package.json

```
@@ -5,7 +5,7 @@
     "start": "yarn docs:dev",
     "docs:dev": "yarn workspace docs dev",
     "docs:api": "yarn docs:api:build",
-    "docs:api:build": "cross-env BABEL_ENV=development babel-node --config-file ./docs/babel.config.js -i '/node_modules/(?!@material-ui)/' -x .ts,.tsx,.js ./docs/scripts/buildApi.ts ./docs/pages/api-docs/data-grid",
+    "docs:api:build": "cross-env BABEL_ENV=development babel-node --config-file ./docs/babel.config.js -i \"/node_modules/(?!@material-ui)/\" -x .ts,.tsx,.js ./docs/scripts/buildApi.ts ./docs/pages/api-docs/data-grid",
```

This seems to work too.

```suggestion
    "docs:api:build": "cross-env BABEL_ENV=development babel-node -x .ts,.tsx,.js ./docs/scripts/buildApi.ts ./docs/pages/api-docs/data-grid",
```

---

Ok this works for me as well, but better if @m4theushw can confirm with this suggested change.

---

Yes, it works. The reason to not ignore node_modules was because, at the time the script was made, one of the imports was using ES6. Now this file uses the CJS syntax.

https://github.com/mui-org/material-ui/blob/f60126f95d7b36fa12133887990615b9f47e994a/docs/src/modules/utils/parseMarkdown.js#L5